src/dos/dos.cpp

```diff
@@ -148,7 +150,52 @@ static uint16_t DOS_GetAmount(void) {
 #ifndef DOSBOX_CPU_H
 #include "cpu.h"
 #endif
-static inline void modify_cycles(Bits value) {
+
+// Taken from Dosbox-X
+int hdd_data_rate = 0;
+int fdd_data_rate = 0;
+
+void DOS_SetDataRate(int rate, DiskType type)
```

Comment:
Let's call this `DOS_SetDiskSpeed()` because that's what it's doing.

`rate` doesn't tell the reader anything about what the number represents; it must be self-descriptive. I suggest `rate_kbyte_per_sec`, that's 100% unambigous. All other `rate` params in the code need to be renamed accoringly everywhere (so `hdd_rate_kbyte_per_sec`, etc.)